src/components/NewBranchDialog.tsx

```diff
@@ -278,7 +279,7 @@ export class NewBranchDialog extends React.Component<
         key={branch.name}
         onClick={this._onBranchClickFactory(branch.name)}
       >
-        <span className={classes(listItemIconClass, 'jp-Icon-16')} />
+        <branchIcon.react className={listItemIconClass} tag="span" />
```

Comment:
Athan made a convincing argument (on the basis of performance) as for why the branch icon should be a background image and not an inline svg here. What's the impetus for changing it now?

ping @kgryte

Reply:
I think @fcollonval was trying to be consistent, but IMO we should sacrifice dev consistency for performance concerns and revert to using background images where performance concerns are valid, such as here and anywhere where we can expect several duplicate icons (e.g., file list, etc).

Reply:
Yep consistency was the reason. So I will move back to the original code in light of the potential performance issue.